strider/kp_registry.py

```diff
@@ -96,6 +97,10 @@ async def search(
             for desc in WBMT.get_descendants(inverse)
         ]
         object_categories = [desc for cat in object_categories for desc in WBMT.get_descendants(cat)]
+        if settings.openapi_server_maturity == "development":
```

Comment:
It looks like based on the SmartAPI docs, maturity also includes the value "staging." Does that need to be included here?

Reply:
No, this is a pretty quick patch. I'll open an issue where we can revisit how this is done and to add staging if need be.